### DISCUSSION TITLE
CslaExtension for VS 2019

### DISCUSSION THREAD

**Discussion by EstReich** [MarimerLLC/csla#3896] [Questions] [answered]
Is there a version of CslaExtension mentioned [here](https://github.com/MarimerLLC/csla/blob/main/docs/code-generators.md) available for VS 2019?

**Comment by rockfordlhotka** [comment, marked as answer]
All of the generators listed on that page are managed by people outside the CSLA team itself, so you'll need to check with the creator(s) of that particular tool.
